util/envDynawo.sh

```diff
@@ -781,18 +808,37 @@ test_modelica_doc() {
 }
 
 jobs() {
+  if [ ! -x "$DYNAWO_INSTALL_DIR/bin/launcher" ]; then
+    build_3rd_party_version || error_exit
+    config_dynawo || error_exit
+    build_dynawo || error_exit
+  fi
   $DYNAWO_INSTALL_DIR/bin/launcher $@
   RETURN_CODE=$?
   return ${RETURN_CODE}
 }
 
-generate_preassembled() {
+dydLibGenerator() {
+  if [ ! -x "$DYNAWO_INSTALL_DIR/bin/launcher" ]; then
+    build_3rd_party_version || error_exit
+    config_dynawo || error_exit
+    build_dynawo_core || error_exit
+  fi
   $DYNAWO_INSTALL_DIR/bin/launcher --dydlib $*
   RETURN_CODE=$?
   return ${RETURN_CODE}
 }
 
-generate_preassembled_gdb() {
+dydLibGenerator_gdb() {
+  if [ "$BUILD_TYPE" != "Debug" ]; then
+    echo "You should compile with BUILD_TYPE=Debug."
```

Comment:
We can use gdb in release mode. (and I do it on a regular basis)

Reply:
ok